frameworks/nexus/package.json

```diff
@@ -10,15 +10,16 @@
     "dev": "ts-node-dev --no-notify --respawn --transpileOnly src/server"
   },
   "dependencies": {
+    "@nexus/schema": "^0.14.0",
     "@prisma/client": "2.0.0-alpha.1311",
     "graphql": "14.6.0",
     "graphql-yoga": "1.18.3",
-    "nexus": "0.12.0",
-    "nexus-prisma": "0.11.1"
+    "nexus-prisma": "^0.14.0"
   },
   "devDependencies": {
     "@prisma/cli": "2.0.0-alpha.1311",
     "@types/node": "12.12.37",
+    "@types/react": "^16.9.35",
```

Comment:
This is required becuase of https://github.com/graphql-nexus/nexus-schema-plugin-prisma/issues/675